src/Illuminate/Support/Number.php

```diff
@@ -32,7 +32,7 @@ public static function toHuman($number, $locale = 'en')
      */
     public static function bytesToHuman($bytes, $precision = 2)
     {
-        $units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+        $units = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
It's worth noting that Rails uses `KB`. https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html